core/src/test/kotlin/org/stellar/anchor/sep38/Sep38ServiceTest.kt

```diff
@@ -331,112 +351,112 @@ class Sep38ServiceTest {
       )
 
     // empty sell_asset
-    ex = assertThrows { sep38Service.getPrice(getPriceRequestBuilder.build()) }
+    ex = assertThrows { sep38Service.getPrice(sep10Token, getPriceRequestBuilder.build()) }
```

Comment:
nit: Same comment as above, I don't think `sep10Token` is necessary here?

Reply:
These are fixed.